### PR TITLE
NT-MB-10843-bug-fix-portrait-orientation

### PR DESCRIPTION
**Pull request by NamibiaTorres**
## Summary

This ticket fixes the bug that affects the orientation of the Mymove login page. Since the Signin page is shared across application this change will also enable mobile view resizing for all signin pages across application, but only the signin pages. Leveraging the already existing `SignIn.module.scss`, mobile resizing logic was added there since it is already imported and used by `SignIn.jsx`. Mobile view is `375px`, which is set as the min-width of the page.

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

#### In the main branch first
1. `make server_run`
2. `make client_run`
3. Go to localhost:3000 and inpsect the page 
4. Resize the page and note the position of the sign-in button and other features that would adjust to teh resizing of the page
5. Note whether the page  and button resizes accordingly.

#### In the this bug fix branch
1. `make server_run`
2. `make client_run`
3. Go to localhost:3000 and inpsect the page 
4. Resize the page and note the position of the sign-in button and other features that would adjust to teh resizing of the page
5. Note whether the page is adjusting when the page shifts to a mobile view.

### Frontend

- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://transcom.github.io/mymove-docs/docs/about/supported-browsers) (Chrome, Firefox, Edge).
- [ ] There are no new console errors in the browser devtools.
- [ ] There are no new console errors in the test output.
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

### Before:
Screenshot 2023-09-06 at 12 59 31 PM

### After bug fix:
Screenshot 2023-09-06 at 12 59 40 PM

### CODE REVIEW

**Review by dinabeigelman — APPROVED** [review]
lgtm! 🎉

**Review by kctruss — APPROVED** [review]
🚀 Looks good, nice work!

File: src/pages/SignIn/SignIn.jsx [hunk 1]
```diff
@@ -4,7 +4,6 @@ import { bool, shape, string } from 'prop-types';
 import { Button, ButtonGroup } from '@trussworks/react-uswds';
 import { useLocation, useNavigate } from 'react-router-dom';
 
-import '../../styles/office.scss';
```
Comment (ronaktruss) [inline src/pages/SignIn/SignIn.jsx:7]: I'm seeing the following on the customer and office app:
image

You can see that there are some differing styles between the office sign in and the customer sign in.
I'm seeing 3 main differences:
1. The header Welcome text is bold
2. The colors of the buttons differ
3. The spacing between the buttons differ

It appears that on the office signin page we are using the default uswds styles when we should be using our style overrides.  

Additional Info: When we removed this line we also removed the imports to the files in index.scss and ghc_index.scss. Both of these files had their own imports that had additional overrides.

Comment (NamibiaTorres) [reply src/pages/SignIn/SignIn.jsx:7]: I am adding the imports back in. Going to double check that the default styles get overridden with our custom styles.

**Pull request merged**